packages/datadog-instrumentations/src/router.js

```diff
@@ -90,7 +90,7 @@ function createWrapRouterMethod (name) {
 
   function wrapNext (req, next) {
     return function (error) {
-      if (error) {
+      if (error && error !== 'route' && error !== 'router') {
```

Comment:
This should be a check for the `string` type instead, so that it continues to work if additional values are added.

Reply:
That's not how express handles it though. Currently string errors are valid as long as they aren't "route" or "router" so we should mirror that functionality.

Reply:
🤦‍♂️